doc/project/release_process.rst

```diff
@@ -72,9 +57,9 @@ code base has begun.
 Stabilization Phase
 *******************
 
-Over the next weeks and depending on the release milestone, only stabilization,
-cosmetic changes, tests, bug and doc fixes are allowed (See :ref:`table
-<release_milestones>` below).
+Over the next weeks and depending on the release milestone, Only stabilization,
```

Comment:
```suggestion
Over the following weeks and depending on the release milestone, only stabilization,
```

Reply:
yeah well, this text is exactly 10 years old :)